tools/attest/README.md

```diff
@@ -46,3 +46,16 @@ Default value is `bin`.
 Path to output file to write attestation report to.
 
 Default is empty, interpreted as stdout.
+
+
+### `verbose`
+
+If set, then logger appends INFO and WARNING logs to Stdout. Default logger has verbosity set to `0`, so verbosity option should be set to appropriate value to append INFO and WARN logs at variable verbosity levels to Stdout.
+
+Default value is `false`.
+
+### `verbosity`
+
+Used to set the verbosity of logger, where higher number means more verbose output.
+
+Default value is `0`.
```

Comment:
Can you describe the meaning of each verbosity level?

Reply:
Verbosity level is not associated with INFO, WARN, ERROR, FATAL type but instead to the amount of detailed logs we want. E.g.

logger.V(0).Info("detail 1")
logger.V(1).Info("detail 2")
logger.V(2).Info("detail 3")

Verbosity level 1 would show details 1 & 2; level 2 would show details 1, 2 & 3.